### ISSUE TITLE
COMRADES posts are being duplicated upon publish

### ISSUE THREAD

**Issue by Shadrock** [ushahidi/platform#1839]
### Expected behaviour
Publish or submit post with geolocation and have it appear once in map view.

### Actual behaviour
The action of publishing posts seems to have the effect of duplicating the post. [The example post is 2562](https://comrades-stg.ushahidi.com/posts/2562) - screenshot below
<img width="1048" alt="comrades duplicate posts" src="https://user-images.githubusercontent.com/4574998/27709482-7e6cd750-5cd9-11e7-85a1-c5640cef2c33.png">
### Steps to reproduce the behaviour/error

1. Login as administrator. 
2. Select the "Test Survey for COMRADES" survey. 
3. Add a post entering only title, description and location. 
4. Publish post. 

_-- Alternatively --_ 

1. Login as administrator. 
2. Select the "Test Survey for COMRADES" survey. 
3. Select a post that is under review ([a good test might be 2563](https://comrades-stg.ushahidi.com/posts/2563)). 
4. Publish post. 

### Notes
I first saw this issue when _editing_ a post under review, which has since been deleted. The post looked normal and included CREES and YODIE results. The only problem was that the location was wrong. I edited the post by changing the pin on the map and then published the post. An error message at the bottom of the page displayed something to the effect of "multiple locations". I dismissed this error but when I went to the map the same post appeared in _both_ locations: the original, erroneous, location and the new, corrected, location. I deleted the post in the erroneous location only to find this actually deleted both posts. 

When I tried to recreate the post (number 2562 above), I saw that, for some strange reason, it showed up twice in the same location. In this case I published _immediately_ upon making the post: CREES and YODIE callbacks had not been received. However, in the first case (trying to publish the existing post) both CREES and YODIE callbacks had been completed and the information was displayed.

**Comment by willdoran** [comment]
Fixed in Service Proxy

**Comment by Shadrock** [comment]
This is still an issue. I made a new post this morning (number 2568), waited until YODIE and CREES callbacks were completed, reviewed, and then published the post. I received the error message (see screenshot below) and when I dismissed it, found duplicate posts in map view: the duplicates do not appear in Timeline view, only map view. 
![screen shot 2017-06-30 at 11 10 58 am](https://user-images.githubusercontent.com/4574998/27746657-4f6df98c-5d85-11e7-8e84-e8a270d8ff2e.png)
